ga4gh/datamodel/reads.py

```diff
@@ -189,6 +191,34 @@ def toProtocolElement(self):
         stats.baseCount = None  # TODO requires iterating through all reads
         readGroup.stats = stats
         readGroup.programs = self.getPrograms()
+        experiment = protocol.Experiment()
+        # required experiment fields
+        experiment.id = 'TODO'
```

Comment:
Can we make a hierarchical ID for experiment? We can just tack and 'experiment' on the end to distinguish it from the ReadGroup id.